modules/get-started/pages/quick-start.adoc

```diff
@@ -36,111 +55,343 @@ docker compose version
+message.timestamp.type                CreateTime  DEFAULT_CONFIG
+redpanda.iceberg.delete               true        DEFAULT_CONFIG
+redpanda.iceberg.mode                 disabled    DEFAULT_CONFIG
+redpanda.leaders.preference           none        DEFAULT_CONFIG
+redpanda.remote.delete                true        DEFAULT_CONFIG
+redpanda.remote.read                  true        DEFAULT_CONFIG
+redpanda.remote.write                 true        DEFAULT_CONFIG
+retention.bytes                       -1          DEFAULT_CONFIG
+retention.local.target.bytes          -1          DEFAULT_CONFIG
+retention.local.target.ms             86400000    DEFAULT_CONFIG
+retention.ms                          604800000   DEFAULT_CONFIG
+segment.bytes                         134217728   DEFAULT_CONFIG
+segment.ms                            1209600000  DEFAULT_CONFIG
+write.caching                         true        DEFAULT_CONFIG
 ----
+====
 
---
-======
+=== Data transforms
+
+Data transforms let you run common data streaming tasks on the Redpanda broker, like filtering, scrubbing, and transcoding. For example, you may have consumers that require you to redact credit card numbers or convert JSON to Avro.
+
+This deployment comes with one transform function called `regex` running in your cluster. Its job is to find records in the `logins` topic that contain email addresses with the `.edu` domain and add those to a new topic called `edu-filtered-domains`.
+
+In the left menu, click *Transforms*.
+// (step {"action":"find", "selector": "[data-testid='nav-link-Transforms']", "matchText": "Transforms", "click": true})
+
+On the *Transforms* page, you see your transform. You can use Redpanda Console to manage and monitor your transforms.
+// (step {"action":"wait"})
+// (step {"action":"saveScreenshot", "path": "transforms.png", "directory": "../../modules/console/images", "overwrite": "byVariation"})
+
+NOTE: The source code for the data transform function is in the `docker-compose/transform/transform.go` file that you downloaded. This file is commented to explain how it works.
+
+image::console:transforms.png[]
+
+See also:
+
+- xref:console:ui/data-transforms.adoc[]
+- xref:develop:data-transforms/index.adoc[]
+
+// (step {"action":"goTo", "url": "http://localhost:8080/admin/upload-license"})
+// (step {"action":"wait"})
+// (step {"action":"saveScreenshot", "path": "license.png", "directory": "../../modules/console/images", "overwrite": "byVariation"})
 
 == Start streaming
 
-To start building a basic streaming application, you can use the `rpk` command-line tool to create a topic, produce messages to it, and consume messages from it.
+To start building a basic streaming application, you can use the Redpanda CLI (`rpk`) to create a topic, produce messages to it, and consume messages from it. Each Redpanda broker comes preinstalled with `rpk`, so you can use it inside one of the Redpanda broker's Docker containers.
```

Comment:
I think you should intro this section by saying that you can do all of this in either Redpanda Console or in rpk. Maybe even show example of one of the steps doing it both ways.